lib/LaTeXML/Post/Manifest/Epub.pm

```diff
@@ -226,7 +226,7 @@ sub finalize {
         my $OPS_abspath  = $_;
         my $OPS_pathname = pathname_relative($OPS_abspath, $OPS_directory);
         my (undef, $name, $ext) = pathname_split($OPS_pathname);
-        if (-f $OPS_abspath && $ext ne 'xhtml' && "$name.$ext" ne 'LaTeXML.cache') {
+        if (-f $OPS_abspath && $ext ne 'xhtml' && "$name.$ext" ne 'LaTeXML.cache' && $OPS_abspath ne 'content.opf') {
```

Comment:
This fixes an unintended behaviour from #1639. So much for submitting a PR without testing every single case. Without this fix, if the manifest `content.opf` is supplied by the user, it ends up referenced within itself with type `application/octet-stream`. Not sure what consequences it would have, but better not to do that.